packages/wallets/src/metamask/metamask.page.ts

```diff
@@ -61,6 +61,24 @@ export class MetamaskPage implements WalletPage {
     });
   }
 
+  async cancelAllTxInQueue() {
+    await test.step('Force close all transactions', async () => {
+      await test.step('Close all extension pages', async () => {
+        const pages = await this.browserContext.pages();
+        await Promise.all([
+          pages.map((page) => {
+            if (page.url().startsWith('chrome-extension://')) page.close();
```

Comment:
mayb you can reuse or contains(this.config.COMMON.STORE_EXTENSION_ID) instead of chrome-extension://

Reply:
fixed